### PR TITLE
Merg this.  (then delet this)

### LINKED ISSUE #88: Delete the meme when the message used to create the meme was deleted

### PR DESCRIPTION
**Pull request by ironchefpython**
fixes #88

### CODE REVIEW

File: nachomemes/__init__.py [hunk 1]
```diff
@@ -9,6 +10,23 @@
 from .local_store import LocalTemplateStore
 from .render import render_template
 
+def get_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(
+        description="Runs the bot with the passed in arguments."
+    )
+
+    parser.add_argument(
+        "-d",
```
Comment (jswny) [inline nachomemes/__init__.py:19]: LOL formatter moment!

Comment (ironchefpython) [reply nachomemes/__init__.py:19]: yeah

File: nachomemes/__init__.py [hunk 2]
```diff
@@ -9,6 +10,18 @@
 from .local_store import LocalTemplateStore
 from .render import render_template
 
+def get_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(
+        description="Runs the bot with the passed in arguments."
+    )
+
+    parser.add_argument("-d", "--debug", action="store_true",
+        help="Run state::debug. True or false. Runs different credentials and logging level.")
```
Comment (jswny) [inline nachomemes/__init__.py:19]: WTF is `state::debug`?

File: pyproject.toml [hunk 3]
```diff
@@ -23,6 +23,7 @@ pylint = "^2.7.2"
 autopep8 = "^1.5.5"
 yamllint = "^1.26.0"
 mypy = "^0.812"
+rope = "^0.18.0"
```
Comment (jswny) [inline pyproject.toml:26]: I hate that VSCode does this... but IDK if there's a way to fix it?

**Review by jswny — APPROVED** [review]
Poggers

File: pyproject.toml (hunk 3 above)
Comment (ironchefpython) [reply pyproject.toml:26]: delete it?

File: nachomemes/__init__.py (hunk 2 above)
Comment (ironchefpython) [reply nachomemes/__init__.py:19]: I don't know

File: pyproject.toml (hunk 3 above)
Comment (jswny) [reply pyproject.toml:26]: Mine doesn't do this... you delete it

**Pull request merged**